graphql/database/users/__tests__/getShouldShowYahooPrompt.test.js

```diff
@@ -50,17 +68,47 @@ describe('shouldShowYahooPrompt tests', () => {
     expect(result).toEqual(false)
   })
 
-  it('returns true if users feature is true for test', async () => {
+  it('returns true if users feature is true for existing user test', async () => {
     expect.assertions(1)
     const getShouldShowYahooPrompt = require('../getShouldShowYahooPrompt')
       .default
-    getUserFeature.mockResolvedValueOnce(
-      new Feature({
-        featureName: YAHOO_SEARCH_EXISTING_USERS,
-        variation: true,
-        yahooPaidSearchRewardOptIn: false,
-      })
-    )
+    getUserFeature
+      .mockResolvedValueOnce(
+        new Feature({
+          featureName: YAHOO_SEARCH_EXISTING_USERS,
+          variation: true,
+          yahooPaidSearchRewardOptIn: false,
+        })
+      )
+      .mockResolvedValueOnce(
+        new Feature({
+          featureName: YAHOO_SEARCH_NEW_USERS,
+          variation: 'Google',
+        })
+      )
+
+    const result = await getShouldShowYahooPrompt(userContext, user)
+    expect(result).toEqual(true)
+  })
+
+  it('returns true if users feature is true for new user test', async () => {
+    expect.assertions(1)
+    const getShouldShowYahooPrompt = require('../getShouldShowYahooPrompt')
+      .default
+    getUserFeature
+      .mockResolvedValueOnce(
+        new Feature({
+          featureName: YAHOO_SEARCH_EXISTING_USERS,
+          variation: findLastIndex,
```

Comment:
What's the effect of `findLastIndex` here?

Reply:
No idea what was going on here. Will update real quick.